src/libraries/view-annotations/Hyperlink.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FunctionComponent, PropsWithChildren } from 'react';
+
+type Props = {
+    onClick?: () => void
+    href?: string
+    target?: string
+    style?: React.CSSProperties
+}
```

Comment:
The `Hyperlink` component seems pretty foundational, but it's already defined with the same definition as this one in `libraries/view-experimental-selector-1/Hyperlink.tsx` and `view-timeseries-graph/Hyperlink.tsx`. I think we should use one definition--I think we get more benefit from making it easy to be consistent than making it easy to redefine this, especially when we don't have any particular local customizations in mind for different consumer components.

Reply:
done